GPU/Directx9/VertexShaderGeneratorDX9.cpp

```diff
@@ -232,7 +232,7 @@ void GenerateVertexShaderDX9(int prim, char *buffer, bool useHWTransform) {
 			WRITE(p, "%s", boneWeightAttrDecl[TranslateNumBones(vertTypeGetNumBoneWeights(vertType))]);
 		}
 		if (doTexture && hasTexcoord) {
-			if (doTextureProjection)
+			if (doTextureProjection && !throughmode)
```

Comment:
Sorry, yeah, you are right, it's different. However, it appears to not cause too many issues :) But yes, that it's different is clearly a bug.